### ISSUE TITLE
Label: A11y Query: Is "colon" supposed to be read by screenreaders

### ISSUE THREAD

**Issue by abhijeet181088** [SAP/ui5-webcomponents#9195]
### Describe the bug

This is a consulting question. 

When showColon={true} param is passed to the Label component, it adds colon after the label. Are the screenreaders supposed to read it?

I see aria-hidden="true" for the colon, which makes sense. Just want to make sure that it is intentional and as per a11y guidelines.

**As per our accessibility testing team, in reading mode, even ":" should be read, which is not the case here.**

### Isolated Example

https://stackblitz.com/edit/github-myzwsj?file=src%2FApp.tsx

### Reproduction steps

Query mentioned in description.

### Expected Behaviour

Consulting question.

### UI5 Web Components for React Version

1.29.0

### UI5 Web Components Version

1.24.0

### Browser

Chrome

### Operating System

macOS

### Organization

SAP

### Declaration

- [X] I’m not disclosing any internal or sensitive information.
